config/routes.rb

```diff
@@ -61,15 +61,15 @@
     end
   end
 
-  resources :profile, except: [:show, :new, :create, :edit, :update, :destroy]
+  resources :profile, only: :index
 
-  resources :search, except: [:show, :new, :create, :edit, :update, :destroy] do
+  resources :search, only: :index do
     collection do
       get 'posts'
     end
   end
 
-  resources :notifications, except: [:show, :new, :create, :edit, :update] do
+  resources :notifications, only: :destroy do
```

Comment:
The NotificationsController does use the index route as a fallback for the redirect_back when rendering an HTML request.

```ruby
  def destroy
    @notification.destroy if @notification.present?

    respond_to do |format|
      format.html { redirect_back(fallback_location: notifications_path) }
      format.json { head :no_content }
    end
  end
```
As such this should be 
```suggestion
  resources :notifications, only: [:destroy, :index] do
```

Reply:
@ludamillion Ah I see. Thanks for letting me know.

Let me update my PR.